### PR TITLE
[WIP] Containers Policies first commit

### PR DESCRIPTION
**Pull request by pavelzag**
Purpose or Intent
=================
{{pytest: cfme/tests/containers/test_containers_policy.py -v --use-provider cm-env2 }}

This test is testing Policy and Policy Profiles assignments on Containers

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 43adbf10fe3af51ef7562e8d1922842a6c3d23df:

Lint OK! :cake: :punch: :cookie: .


**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

File: cfme/tests/containers/test_containers_policy.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+def control_policy(request):
+    scope = explorer.ReplicatorControlPolicy(
+        fauxfactory.gen_alphanumeric(),
+        scope="fill_count(Replicator.Pods, >, 1)"
+
+    )
+    scope.create()
+    yield scope
+
+
```
Comment (gshefer) [inline cfme/tests/containers/test_containers_policy.py]: polarion id?

Comment (pavelzag) [inline cfme/tests/containers/test_containers_policy.py]: This PR dates a few months back :D Will update later

**Comment by jkrocil** [comment]
@pavelzag Is this still under active development?

**Comment by pavelzag** [comment]
@jkrocil Not very active at the moment. Will reevaluate it in the upcoming days

**Comment by jkrocil** [comment]
Feel free to reopen once you spend some more time on this. Thanks!

**Pull request closed without merging**